themes/bubble-wrap-icons-shields-international.yaml

```diff
@@ -0,0 +1,2140 @@
+            'UA:national-6char': [198, 668, 76, 36]
+            'UA:regional-6char': [198, 668, 76, 36]
+            'IT:SP-6char': [198, 668, 76, 36]
+            'IT:SR-6char': [198, 668, 76, 36]
+            'IT:SS-6char': [198, 668, 76, 36]
+            'CZ:regional-6char': [198, 668, 76, 36]
+            'UA:international-7char': [108, 668, 82, 36]
+            'UA:national-7char': [108, 668, 82, 36]
+            'UA:regional-7char': [108, 668, 82, 36]
+            'IT:SP-7char': [108, 668, 82, 36]
+            'IT:SR-7char': [108, 668, 82, 36]
+            'IT:SS-7char': [108, 668, 82, 36]
+            'CZ:regional-7char': [108, 668, 82, 36]
+            'ZA:metropolitan-1char': [378, 388, 36, 54]
+            'ZA:metropolitan-2char': [334, 390, 38, 50]
+            'ZA:metropolitan-3char': [282, 390, 46, 50]
+            'ZA:metropolitan-4char': [222, 390, 54, 50]
+            'ZA:metropolitan-5char': [156, 388, 60, 52]
+            'ZA:national-1char': [322, 448, 46, 44]
+            'ZA:national-2char': [266, 448, 50, 44]
+            'ZA:national-3char': [200, 448, 58, 44]
+            'ZA:national-4char': [128, 448, 64, 44]
+            'ZA:national-5char': [50, 448, 74, 44]
+            'ZA:provincial-1char': [0, 448, 46, 50]
+            'ZA:provincial-2char': [920, 390, 50, 50]
+            'ZA:provincial-3char': [856, 390, 58, 50]
+            'ZA:provincial-4char': [780, 390, 70, 50]
+            'ZA:provincial-5char': [696, 390, 78, 50]
+            'ZA:regional-1char': [654, 390, 36, 38]
+            'NO:riksvei-1char': [654, 390, 36, 38]
+            'RO:motorway-1char': [654, 390, 36, 38]
+            'ZA:regional-2char': [604, 390, 42, 38]
+            'NO:riksvei-2char': [604, 390, 42, 38]
+            'RO:motorway-2char': [604, 390, 42, 38]
+            'ZA:regional-3char': [550, 390, 48, 38]
+            'NO:riksvei-3char': [550, 390, 48, 38]
+            'RO:motorway-3char': [550, 390, 48, 38]
+            'ZA:regional-4char': [488, 390, 56, 38]
+            'NO:riksvei-4char': [488, 390, 56, 38]
+            'RO:motorway-4char': [488, 390, 56, 38]
+            'ZA:regional-5char': [420, 390, 62, 38]
+            'NO:riksvei-5char': [420, 390, 62, 38]
+            'RO:motorway-5char': [420, 390, 62, 38]
+            'ru:national-1char': [0, 720, 42, 36]
+            'RU:national-1char': [0, 720, 42, 36]
+            'GB:M-road-1char': [0, 720, 42, 36]
+            'GR:GR-1char': [0, 720, 42, 36]
+            'GR:national-1char': [0, 720, 42, 36]
+            'IT:SR-1char': [0, 720, 42, 36]
+            'IT:SS-1char': [0, 720, 42, 36]
+            'IT:B-road-1char': [0, 720, 42, 36]
+            'ES:A-road-1char': [0, 720, 42, 36]
+            'SE:A-1char': [0, 720, 42, 36]
+            'AT:A-road-1char': [0, 720, 42, 36]
+            'AT:S-road-1char': [0, 720, 42, 36]
+            'TR:highway-1char': [0, 720, 42, 36]
+            'LU:M-1char': [0, 720, 42, 36]
+            'FI:LR-1char': [0, 720, 42, 36]
+          
```

Comment:
No need for `all` here, see comments above.

Reply:
336ba5a6078c7b400ae2b061392b0690340cddcf